eng/source-build-patches/0026-Pin-System.Security.Cryptography.Cng-to-single-versi.patch

```diff
@@ -0,0 +1,30 @@
+From 82678ff2bec86be98d9797f98dab99aa1e82c25c Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Fri, 30 Oct 2020 16:25:21 +0000
+Subject: [PATCH] Pin System.Security.Cryptography.Cng to single version
+
+If System.Security.Cryptography.Cng is restored using the PVP
+version (5.0.0), this causes conflicts later in the restore which
+results in an error:
+   An item with the same key has already been added.
+   Key: (System.Security.Cryptography.Cng, 5.0.0)
+---
+ .../src/System.Security.Cryptography.Pkcs.csproj                | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/libraries/System.Security.Cryptography.Pkcs/src/System.Security.Cryptography.Pkcs.csproj b/src/libraries/System.Security.Cryptography.Pkcs/src/System.Security.Cryptography.Pkcs.csproj
+index e9db368ef24..c396c21a798 100644
+--- a/src/libraries/System.Security.Cryptography.Pkcs/src/System.Security.Cryptography.Pkcs.csproj
++++ b/src/libraries/System.Security.Cryptography.Pkcs/src/System.Security.Cryptography.Pkcs.csproj
+@@ -639,7 +639,7 @@
+     <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime.CompilerServices.Unsafe\src\System.Runtime.CompilerServices.Unsafe.ilproj" />
+   </ItemGroup>
+   <ItemGroup Condition="!$(TargetFramework.StartsWith('$(NetCoreAppCurrent)'))">
+-    <PackageReference Include="System.Security.Cryptography.Cng" Version="$(SystemSecurityCryptographyCngVersion)" />
++    <PackageReference Include="System.Security.Cryptography.Cng" Version="4.7.0" />
```

Comment:
Same here. The package is dead-ended and the last version is 5.0.0 and should be used. The patch should be removed.

Reply:
This patch has been removed, compliments of you!  Thanks